test/parallel/test-regress-GH-13435.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
```

Comment:
Maybe shorten a bit to `test-cluster-send-socket-to-worker-server`, but it's up to you.